.github/workflows/windows-release-build.yml

```diff
@@ -32,6 +32,6 @@ jobs:
         run: |
           ARCHIVE=php_mongodb-${{ github.ref_name }}-${{ matrix.php }}-${{ matrix.ts }}-${{ matrix.arch }}.zip
           zip $ARCHIVE php_mongodb.dll php_mongodb.pdb CREDITS CONTRIBUTING.md LICENSE README.md THIRD_PARTY_NOTICES
-          hub release edit -a $ARCHIVE -m "" $GITHUB_REF_NAME
+          gh release upload $GITHUB_REF_NAME $ARCHIVE
```

Comment:
`hub` is no longer available in runners, as evidenced by [this CI failure](https://github.com/mongodb/mongo-php-driver/actions/runs/7105117545/job/19341706486#step:4:16). I'm going to manually attached the release assese to 1.17.1 in the meantime, but this should fix things going forward.